lib/meetup.ts

```diff
@@ -0,0 +1,67 @@
+export async function getEvents() {
+  const gql = `
+query {
+  groupByUrlname(urlname: "devICT") {
+    upcomingEvents(input:{first: 10, reverse: true}) {
+      edges {
+        node {
+          title
+          dateTime
+          shortUrl
+        }
+      }
+    }
+  }
+}
+`.trim();
+
+  const respData = await fetch("https://api.meetup.com/gql", {
+    method: "POST",
+    body: JSON.stringify({ query: gql }),
+  }).then((r) => r.json());
```

Comment:
It's generally best practice to check if the response was >= 400 status, meaning the request failed in some way.

```suggestion
  const resp = await fetch("https://api.meetup.com/gql", {
    method: "POST",
    body: JSON.stringify({ query: gql }),
  })

  if (!resp.ok) {
    throw new Error(`meetup request failed: ${await resp.text()}`);
  }

  const respData = await resp.json();
```